getting-started/pull-request-lifecycle.rst

```diff
@@ -487,14 +487,55 @@ Instead of simply "approving" the pull request, leave comments.  For example:
 #. Comment on what is "good" about the pull request, not just the "bad". Doing
    so will make it easier for the PR author to find the good in your comments.
 
+#. Look at any failures in CI on the current PR. See :ref:`"Keeping CI green"
+   <keeping-ci-green>` below for simple things you can do to help move the PR forward.
+
 Dismissing review from another core developer
-=============================================
+---------------------------------------------
 
 A core developer can dismiss another core developer's review if they confirmed
 that the requested changes have been made.  When a core developer has assigned
 the PR to themselves, then it is a sign that they are actively looking after
 the PR, and their review should not be dismissed.
 
+.. _keeping-ci-green:
+
+Keeping Continuous Integration green
```

Comment:
```suggestion
Keeping continuous integration green
```

Reply:
but that's CI, a proper name, like GitHub?

Reply:
No, it's a common noun.

If short for continuous integration server/service, there are many named ones: Azure Pipelines, Jenkins, Gitlab Pipelines, AppVeyor. The proper noun of a CI service we're using is GitHub Actions.

If continuous integration as the name for a practice, it's also a common noun. For example, see https://en.wikipedia.org/wiki/Continuous_integration